CHANGELOG.md

```diff
@@ -5,14 +5,27 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-
 ## [Unreleased]
 ### Added
 
 ### Fixed
 
 ### Changed
 
+## [0.2.13] - 2024-05-24
+
+### Added
+
+- Added classes for the pattern commands and circuit instructions. It has been done for data validation, readability and maintainability purposes. All the commands and instructions were represented in raw inside lists which allowed wrong datas passed, but also made it difficult to turn the code into a more general way to build commands and instructions.
+  - Added `class Command` and all its child classes that represent all the pattern commands.
+  - Added `class Instruction` for the quantum circuit model. Every instruction can be instanciated using this class by passing its name as defined in the Enum `InstructionName`.
+
+### Fixed
+
+### Changed
+
+- Made all the updates inside the files making use of the commands and instructions list representation.
```

Comment:
You should rephrase the change in terms of API changes: the user cannot write `pattern.add(["M", 0, "XY", 0, [], []])` anymore but should write `pattern.add(M(node=0))` for instance.